forgot_password.html

```diff
@@ -0,0 +1,48 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=0.9">
+    <title>Forgot password</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons&display=block">
+    <link rel="stylesheet" href="styles/style.css">
+    <link href="styles/normalize.css" rel="stylesheet">
+</head>
+<body id="forgot_password_page">
+<div class="container-fluid">
+    <div class="row" id="logo_invoicer_big_forgot_password_page">
+        <img src="img/logo.svg" width="412" height="146"/>
+    </div>
+    <div class="row text-center">
+        <p id="forgot_password_text">Restore password</p>
+    </div>
+    <div class="row text-center">
+        <p id="confirmation_text">Confirmation of changing password was sent to <span
+                id="user_email_forgot_password_page">{user_email}</span></p>
```

Comment:
id for span don't need. You can use selector by elements with a specific parent. For instance `p > span`

Reply:
We have used this id, to manipulate with content of the page using JS

`    const span_user_email = document.getElementById("user_email_forgot_password_page");
`
We don't use it in styles.
Should we remove it?

Reply:
you also could use css selectors in java scirpt. Just use method `document.querySelector()` [link](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector)

Reply:
Thank you for hint! Quite useful! 
Done